DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -272,19 +273,19 @@ function Set-TargetResource
             {
                 Set-AuthenticationInfo -Site $Name -AuthenticationInfo $AuthenticationInfo -ErrorAction Stop
             }
-           
+
             # Update Preload if required
             if ($PSBoundParameters.ContainsKey('preloadEnabled') -and $Website.applicationDefaults.preloadEnabled -ne $PreloadEnabled)
```

Comment:
While you're correcting the layout of this module, would you mind fixing these lines that go over the 100 char limit?

Shamless plug here: I wrote a blog post on using vertical guides to identify lines over 100 chars - [here](https://dscottraynsford.wordpress.com/).